screenpy/resolutions/custom_matchers/is_in_bounds.py

```diff
@@ -22,8 +22,17 @@
     InequalityFunc = Callable[[float, float], bool]
 
 
+class DegenerateIntervalError(ValueError):
+    """ValueError specifically for degenerate intervals."""
+
+
 class IsInBounds(BaseMatcher[float]):
-    """Matches a number which is in the given range."""
+    """
+    Matches a number which is in the given range.
```

Comment:
This first line should still be level with the `"""`. Also i don't think we can have the `.` at the end of the URL like that.

(Well by jove, we can, but it breaks the link-to-the-heading.)

```python
    """Matches a number which is in the given range.

    Only supports proper intervals.
    https://en.wikipedia.org/wiki/Interval_%28mathematics%29#Notations_for_intervals
```

Reply:
I think that was done automatically by the `pydocstyle` rules.  I would never have done that myself. 
Interestingly enough, when I run the rules now it doesn't seem to add the period.  I'm guessing that got fixed.

Reply:
Sorry i'm coming back to this so late, but if you move the sentence back up to the `"""`s, does linting move it back down again?

Reply:
Fixed.  Latest version of ruff isn't trying to reformat the first line anymore.